metadata-ingestion/src/datahub/configuration/validate_field_rename.py

```diff
@@ -22,10 +23,12 @@ def _validate_field_rename(cls: Type, values: dict) -> dict:
                     f"Cannot specify both {old_name} and {new_name} in the same config. Note that {old_name} has been deprecated in favor of {new_name}."
                 )
             else:
-                warnings.warn(
-                    f"The {old_name} is deprecated, please use {new_name} instead.",
-                    UserWarning,
-                )
+                if print_warning:
+                    warnings.warn(
```

Comment:
Do we already set logging.captureWarnings(TRUE) in our logging settings? See [this](https://docs.python.org/3/library/logging.html#logging.captureWarnings).

Reply:
We don't, and I don't think we want to - for the end user, the message gets printed out in a clearer way with warnings. Also, not capturing lets us make assertions based on warnings in tests

Reply:
How about UI based ingestion?